packages/scripts/src/commands/app-data/postProcess/assets.ts

```diff
@@ -63,6 +64,7 @@ export class AssetsPostProcessor {
     fs.ensureDirSync(appAssetsFolder);
     // Generate a list of all deployment assets, merge with list of assets from parent
     const sourceAssets = generateFolderFlatMap(sourceAssetsFolder, { includeLocalPath: true });
+
```

Comment:
Code tidy: delete this line. Not important but it would make the PR neater because this wouldn't show as a difference